File: go_company/api/includes/type/db/company_thread/user_inbox.go
package company_thread

// -------------------------------------------------------
// пакет, содержащий интерфейсы для работы с таблицей
// содержащей непрочитанные пользователя в тредах
// -------------------------------------------------------

import (
	"context"
	"database/sql"
	"fmt"
	"github.com/getCompassUtils/go_base_frame/api/system/mysql"
)

const tableKey = "user_inbox"

type UserInboxRow struct {
	UserId             int64
	MessageUnreadCount int
}

// GetOne получить запись
func (dbConn *DbConn) GetOne(ctx context.Context, userId int64) (*UserInboxRow, error) {

	// запрос проверен на EXPLAIN (INDEX=PRIMARY).
	query := fmt.Sprintf("SELECT user_id, message_unread_count FROM `%s` WHERE `user_id` = ? LIMIT %d", tableKey, 1)

	userInboxRow := &UserInboxRow{}

	queryCtx, cancel := context.WithTimeout(ctx, mysql.QueryTimeout)
	defer cancel()

	err := dbConn.Conn.QueryRowContext(queryCtx, query, userId).Scan(&userInboxRow.UserId, &userInboxRow.MessageUnreadCount)
	if err != nil {

		if err == sql.ErrNoRows {
			return nil, nil
		}
		return nil, err
	}

	return userInboxRow, nil
}

File: php_monolith/src/Compass/Conversation/api/includes/gateway/db/company_conversation/user_inbox.php
<?php

namespace Compass\Conversation;

/**
 * класс-интерфейс для таблицы company_conversation.user_inbox
 */
class Gateway_Db_CompanyConversation_UserInbox extends Gateway_Db_CompanyConversation_Main {

	protected const _TABLE_KEY = "user_inbox";

	// создание записи
	public static function insert(array $insert):void {

		static::_connect(self::_getDbKey())->insert(self::_getTable(), $insert);
	}

	// получение записи
	public static function getOne(int $user_id):array {

		$query = "SELECT * FROM `?p` WHERE `user_id` = ?i LIMIT ?i";
		return static::_connect(self::_getDbKey())->getOne($query, self::_getTable(), $user_id, 1);
	}

	// обновление записи
	public static function set(int $user_id, array $set):void {

		// запрос проверен на EXPLAIN (INDEX=`PRIMARY`)
		$query = "UPDATE `?p` SET ?u WHERE `user_id` = ?i LIMIT ?i";
		static::_connect(self::_getDbKey())->update($query, self::_getTable(), $set, $user_id, 1);
	}

	// -------------------------------------------------------
	// PROTECTED METHODS
	// -------------------------------------------------------

	// получаем таблицу
	protected static function _getTable():string {

		return self::_TABLE_KEY;
	}
}